examples/deployer/src/index.ts

```diff
@@ -38,18 +38,19 @@ async function main() {
 
   // Connect it with a Provider (https://docs.ethers.org/v5/api/providers/)
   const network = "goerli";
```

Comment:
side note: we could probably start moving these examples all over to `sepolia` . totally unrelated to this PR; feel free to leave as is!